.gitlab-ci.yml

```diff
@@ -43,6 +43,7 @@ before_script:
   - eval $(opam env)
   - opam list
   - opam config list
+  - dune printenv
```

Comment:
Please pass the `--root .` option so we are more robust in the unlikely case that there is a `dune-project` file in some upper directory.